supported-targets.md

```diff
@@ -62,6 +74,8 @@
 | Nordic Semi       | nRF91       | ✔ Full        | v1.10.0            | nrf91.c        | -
 | Texas Instruments | MSP432P4    | ✔ Full        | v1.6.2             | msp432p4.c     | -
 | Texas Instruments | MSP432E4    | ✔ Full        | v1.10.0            | msp432e4.c     | -
+| Texas Instruments | MSPM0       | ✔ Full        | v2.0.0             | MSPM0.c        | -
```

Comment:
The file name is lower case - we consider them to be case sensitive as most filesystems consider them to be case sensitive.